### PR TITLE
[WIP] Add reference for issue with rootdir

### PR DESCRIPTION
**Pull request by erheron**
Ref. #6376 
I was thinking about adding some example, but maybe just referencing some "common" issue would help?

### CODE REVIEW

File: doc/en/customize.rst [hunk 1]
```diff
@@ -20,7 +20,9 @@ which were registered by installed plugins.
 Initialization: determining rootdir and inifile
 -----------------------------------------------
 
-pytest determines a ``rootdir`` for each test run which depends on
+Pytest defines ``rootdir`` parameter which is determined for each test run.
+It should't be consufed with ``--rootdir`` command-line argument (please see `https://github.com/pytest-dev/pytest/issues/6359` for example).
```
Comment (hugovk) [inline doc/en/customize.rst]:
```suggestion
It should't be confused with ``--rootdir`` command-line argument (please see `https://github.com/pytest-dev/pytest/issues/6359` for example).
```

File: doc/en/customize.rst [hunk 2]
```diff
@@ -20,7 +20,9 @@ which were registered by installed plugins.
 Initialization: determining rootdir and inifile
 -----------------------------------------------
 
-pytest determines a ``rootdir`` for each test run which depends on
+Pytest defines ``rootdir`` parameter which is determined for each test run.
```
Comment (hugovk) [inline doc/en/customize.rst]:
```suggestion
Pytest defines a ``rootdir`` parameter which is determined for each test run.
```

File: doc/en/customize.rst (hunk 1 above)
Comment (blueyed) [reply doc/en/customize.rst]: > It should't be confused with ``--rootdir`` command-line argument

That sounds wrong: `--rootdir` sets `rootdir` (manually).

You are more likely wanting to explain:

> Unfortunately I agree that the current interaction between rootdir and ini files is far from clear, and someone just looking at --rootdir would assume that it should point to the root of your repository/project, and from there on pytest would find pytest.ini files and start collection, but this is currently not the case.

Comment (erheron) [reply doc/en/customize.rst]: Oops! Wanted to write exactly that, but in shortenned manner :/ Maybe the better option would be re-writing sentence of @nicoddemus like "It shouldn't be confused with the place, from where pytest would find pytest.ini files and start collection (for example,...)"?

**Comment by bluetech** [comment]
I think that `rootdir` is confusing enough, that adding a "should not be confused with ..." would only be more confusing :smile:

I think the current documentation, which just lays out the two thing that `rootdir` is used for, is the best thing to do, short of changing the actual meaning of `rootdir` as discussed in #6943 

Given also that the PR has been open for a while, I'll close it. Thanks @erheron.

**Pull request closed without merging**